sagemaker-python-sdk/pytorch_mnist/pytorch_mnist.ipynb

```diff
@@ -2,9 +2,7 @@
  "cells": [
```

Comment:
<p>How do we fix this?</p>

 _Reply via <a href='https://app.reviewnb.com/jaipreet-s/amazon-sagemaker-examples/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>